paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -960,6 +960,9 @@ class PaymentSheet internal constructor(
             @Parcelize
             sealed class RowStyle : Parcelable {
 
+                internal abstract fun hasSeparators(): Boolean
```

Comment:
Should this be a sealed class instead? `None, Separators(defaultInset: Boolean)`